src/helpers/fallbackAuthHelper.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global console, location, Office, Microsoft */
+
+import { MSGraphHelper } from "./../../node_modules/office-addin-sso/lib/msgraph-helper";
+import { showMessage } from "./../../node_modules/office-addin-sso/lib/message-helper";
+import { writeDataToOfficeDocument } from "./../taskpane/taskpane";
+var loginDialog;
+
+export function dialogFallback() {
+  // We fall back to Dialog API for any error.
+  // TODO: handle specific errors only?
+
+  const url = "/fallbackauthdialog.html";
```

Comment:
Since this file is opened in the dialog and it loads script that runs in the dialog, why is it in a folder named taskpane? I find this confusing.

Reply:
I will go ahead and move this file under src/helpers